python/sources/GooglePubSub/library.json

```diff
@@ -50,11 +50,11 @@
     ],
 	"DeploySettings":
 	{
-		"DeploymentType": "Job",
+		"DeploymentType": "Service",
 		"CpuMillicores": 200,
 		"MemoryInMb": 200,
 		"Replicas": 1,
 		"PublicAccess": false,
-		"ValidateConnection": true
+		"ValidateConnection": false
```

Comment:
does not Google Pub/Sub connector have Connect Validation logic on it?

Reply:
it does